pkg/template/processor_test.go

```diff
@@ -497,6 +482,20 @@ func assertRoleBindingExists(t *testing.T, c client.Client, ns string) authv1.Ro
 	return rb
 }
 
+type decodeFunc func([]byte, *schema.GroupVersionKind, runtime.Object) (runtime.Object, *schema.GroupVersionKind, error)
```

Comment:
this is not used anywhere

Reply:
ah right, that's a left-over :/ I removed it in e165b0b